Functions/Administration/xASL_adm_GetDeprecatedFields.m

```diff
@@ -22,7 +22,7 @@
 
 nameConversionTable = ...
     {'nSessions'                    'dataset'   'nSessions'                          '';...
-    'nSubjectsSessions'             'dataset'   'nSubjectsSessions'                  '';...
+    'nSubjects'                     'dataset'   'nSubjects'                          '';...
```

Comment:
Nice. I just thought that we would have before:

nSessions
nSubjects
nSubjectsSessions

so are you sure that we didn't have nSubjects? Otherwise this could be duplicate.

did you check both nSubjectSessions & nSubjectsSessions?

Reply:
This is all good. I think that you are only confused by this one edit.

This function just takes 
x.nSubjects and puts them into x.dataset.nSubjects

I have checked and `nSubjectSessions` does not exist.
nSubjects was used and is initialized everywhere, we just forgot to put it into this naming correction function. Which wasn't ever a problem, because we never had wrongly placed nSubjects.

Does this answer you?